build/config/linux/detect_host_libc.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+from optparse import OptionParser
+import re
+import subprocess
+import sys
+
+
+def main():
+    if 'linux' not in sys.platform:
+        return 1
+
+    parser = OptionParser()
+    parser.add_option('--compiler-path', action='store',
+                      type='string', default='gcc')
+    (options, args) = parser.parse_args()
+
+    # should work with both gcc and clang
```

Comment:
I'd rather not say this unless we're tesing it with both GCC and Clang. Otherwise it will definitely get broken and become very frustrating for people trying to use it.